AppCenter/AppCenter/Internals/Channel/MSACChannelGroupDefault.m

```diff
@@ -144,7 +144,7 @@ - (void)channel:(id<MSACChannelProtocol>)channel didSetEnabled:(BOOL)isEnabled a
                             }];
 }
 
-- (void)channel:(id<MSACChannelProtocol>)channel didFailSendingLog:(id<MSACLog>)log withError:(NSError *)error {
+- (void)channel:(id<MSACChannelProtocol>)channel didFailSendingLog:(id<MSACLog>)log withError:(nullable NSError *)error {
```

Comment:
Do we need it in `.m` file?

Reply:
It's all optional, but for clarity/readability - yes